test/interop/test_add_function.cpp

```diff
@@ -27,7 +27,7 @@ int main(int argc, char **argv) {
   EM_ASM({
     // Get an export that isn't in the table (we never took its address in C).
     var baz = wasmExports["baz"];
-    var tableSizeBefore = wasmTable.length;
+    var tableSizeBefore = usedTableLength;
```

Comment:
Should use update the `We can reuse indexes` part of the test below too?

Reply:
Sure..

Reply:
Done.